### ISSUE TITLE
ParserError compiling 7.5.0

### ISSUE THREAD

**Issue by jeffblake** [Shopify/polaris-react-archive#4850]
7.4.1 builds and runs just fine, but when I upgrade to 7.5 I get:
<img width="1231" alt="Screen Shot 2021-12-17 at 7 52 58 AM" src="https://user-images.githubusercontent.com/620564/146571703-83ed8280-b61e-4e19-888d-093dd9b483c6.png">

**Comment by mbaumbach** [comment]
I believe this is a duplicate of #4838 and there's an open issue against the `postcss-custom-properties` project. The issue seems to be fixed in the main branch on that project, but a new release is needed. I opened an issue there: https://github.com/postcss/postcss-custom-properties/issues/261

**Comment by mbaumbach** [comment]
There may be a way to workaround this issue with a change in the Polaris styles such that this bug isn't encountered, which might be easier/faster than waiting on the fix in postcss.

**Comment by mbaumbach** [comment]
You might be able to fix this if you upgrade your `postcss-custom-properties` to v12.0.1. That fixed it for me with #4838.

**Comment by jeffblake** [comment]
@mbaumbach that fixed it, thanks. Odd, I check `yarn outdated` frequently but it wasn't reporting those packages were outdated
